src/PhpDoc/TypeNodeResolver.php

```diff
@@ -311,6 +313,14 @@ private function resolveGenericTypeNode(GenericTypeNode $typeNode, NameScope $na
 		}
 
 		$mainType = $this->resolveIdentifierTypeNode($typeNode->type, $nameScope);
+		$broker = Broker::getInstance();
```

Comment:
Can this be injected in the constructor or is there circular reference issue? Anyway, I'd rather have `PHPStan\DependencyInjection\Container` injected into the constructor and the Broker obtained from that using `->getByType`.